src/view/board.js

```diff
@@ -6,7 +6,7 @@ const createBoardTemplate = () => (
 );
 
 
-export default class BoardContainter extends AbstractView {
+export default class BoardContainer extends AbstractView {
```

Comment:
Если на то пошло это вьюха
```suggestion
export default class BoardView extends AbstractView {
```